activities/Moon.activity/js/moon-activity.js

```diff
@@ -319,7 +319,8 @@ define(['activity/data-model', 'activity/draw', 'webL10n', 'sugar-web/env', 'sug
 			file_size: 0
 		};
 		datastore.create(metadata, function() {
-			console.log("export done.")
+            humane.log(webL10n.get('MoonImageSaved'));
```

Comment:
`webL10n` is not define in this context. I guess you should use `l10n`

Reply:
I did the changes the activity just doesn't load up now :( 
I am struggling to find the bug, please help me out here